### DISCUSSION TITLE
I want to edit it and then deploy it so how can I do that?

### DISCUSSION THREAD

**Discussion by VanshajPoonia** [coder/code-server#4660] [Q&A]
I want to edit some of the code and then deploy it. But, if I simply go and deploy it, it will deploy the normal things. I wanna customize it according to me and make it live so can anyone please guide me.

**Comment by lg-nineoneninethreenine** [comment]
I think there's a specific guide on how to do this, like get dependencies, build VSCode, then code-server. I think it's on Coder.com docs

**Reply by VanshajPoonia** [reply]
Can you please provide me with the link if you have?

**Reply by lg-nineoneninethreenine** [reply]
https://coder.com/docs/code-server/latest/CONTRIBUTING

**Reply by VanshajPoonia** [reply]
Thanks a lot! Please help me if I need yours in future.

**Reply by VanshajPoonia** [reply]
just wanna ask that if you are sure that this would work?

**Reply by lg-nineoneninethreenine** [reply]
Yeah although since I don't have any kind of powerhouse machines with me that has 64 GB of RAM I haven't tested building, however, I have installed dependencies and run a FMT on a PR(Gitpod machine). Hoping to make a new build soon on Gitpod!
